packages/core/misc-widgets/src/components/github-star/GithubStar.cy.ts

```diff
@@ -23,4 +23,14 @@ describe('<GithubStar />', () => {
     cy.getTestId('github-star').find('a').should('be.visible')
     cy.getTestId('github-star').find('.github-button').should('be.visible')
   })
+
+  it('renders default tooltip text on focus', () => {
+    cy.mount(GithubStar, {
+      props: {
+        url: 'http://github.com/kong/kong',
+      },
+    })
+    cy.getTestId('github-star').trigger('focus')
+    cy.get('.k-popover').should('exist')
```

Comment:
suggestion: can you check that the tooltip text is present? The popover will always exist in the DOM, so you at least need to check `.should('be.visible')`

Reply:
Added, was having trouble getting the `visible` check to pass, finally `mouseenter` event worked